TI_Practice4/Pactice4.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+
+namespace TI_Practice4
+{
+    class Practice4
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Please write something.");
+            String input = Console.ReadLine();
+
+            Palindrome palindrome1 = new Palindrome();
+            Palindrome palindrome2 = new Palindrome();
+
+            if (input.Length > 1)
+            {
+                if (palindrome1.ToAlphaNumeric(input).Equals(palindrome2.GetInvertedString(input)))
+                {
+                    Console.WriteLine("You have written a palindrome.");
+                }
+                else
+                {
+                    Console.WriteLine("Written string is not a palindrome.");
+                }
+            }
+            else
```

Comment:
the empty string case is not covered in this solution for now

![image](https://user-images.githubusercontent.com/9382934/118269273-3542e480-b4c7-11eb-8116-19b1a010e241.png)